docs/source/index.rst

```diff
@@ -4,11 +4,9 @@ Welcome to HackRF's documentation!
 
 .. toctree::
    :maxdepth: 2
-   :caption: User Documentation
+   :caption: HackRF User Documentation
```

Comment:
I'm confused by the name change to "HackRF User Documentation", especially with Jawbreaker being moved out of this section. It kind of looks like you meant it to be "HackRF One" not "HackRF", but a lot of the content in this section is relevant to all three HackRF platforms, not just HackRF One. The word "HackRF" does not appear in the name of any other section even though they are all directly related to HackRF (except arguably the Opera Cake section).